### PR TITLE
EZEE-2747 Implement minimal test coverage for Copy subtree

### PR DESCRIPTION
**Pull request by katarzynazawada**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZEE-2747
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

**Comment by katarzynazawada** [comment]
@m-tyrala Media folder has Copy subtree button disabled. In Images folder there is a subitem (_Test Article edited4_) added in previous test. Implemented test checks whether it has been copied correctly together with the Images folder.

File: src/lib/Behat/BusinessContext/ContentViewContext.php [hunk 1]
```diff
@@ -74,15 +74,28 @@ public function iSeeTitle(string $title): void
...
+     * @Then there's no :itemName :itemType on :containerName Sub-items list
+     */
+    public function verifyThereIsNoItemInSubItemList(string $itemName, string $itemType, string $containerName): void
+    {
+        $contentItemPage = PageObjectFactory::createPage($this->utilityContext, ContentItemPage::PAGE_NAME, $containerName);
 
         Assert::assertFalse(
             $contentItemPage->subItemList->table->isElementInTable($itemName, $itemType),
-            sprintf('%s "%s" shouldn\'t be on %s Sub-items list', $itemType, $itemName, $folder)
+            sprintf('%s "%s" shouldn\'t be on %s Sub-items list', $itemType, $itemName, $containerName)
```
[the remark this answers is not in the document]
Comment (mnocon) [reply src/lib/Behat/BusinessContext/ContentViewContext.php]: To be honest I like it more this way - these are only two lines and it's very readable, there's no need for any string concatenation and stuff. But it's really a matter of taste here.

[the remark this answers is not in the document]
Comment (katarzynazawada) [reply src/lib/Behat/BusinessContext/ContentViewContext.php]: @m-tyrala, done in https://github.com/ezsystems/ezplatform-admin-ui/pull/944/commits/00f7e63796452c70a13bd64a8352db838cc46f7a

File: src/lib/Behat/BusinessContext/ContentViewContext.php [hunk 2]
```diff
@@ -74,16 +74,39 @@ public function iSeeTitle(string $title): void
     }
 
     /**
-     * @Then there's no :itemName :itemType on :folder Sub-items list
+     * Check if item has or not sub-item, according to $$itemShouldExist param.
```
Comment (mnocon) [inline src/lib/Behat/BusinessContext/ContentViewContext.php]: Double $$

File: src/lib/Behat/BusinessContext/ContentViewContext.php [hunk 3]
```diff
@@ -74,16 +74,39 @@ public function iSeeTitle(string $title): void
...
     {
-        $contentItemPage = PageObjectFactory::createPage($this->utilityContext, ContentItemPage::PAGE_NAME, $folder);
+        $isItemInTable = PageObjectFactory::createPage($this->utilityContext, ContentItemPage::PAGE_NAME, $containerName)
+            ->subItemList->table->isElementInTable($itemName, $itemType);
 
-        Assert::assertFalse(
-            $contentItemPage->subItemList->table->isElementInTable($itemName, $itemType),
-            sprintf('%s "%s" shouldn\'t be on %s Sub-items list', $itemType, $itemName, $folder)
-        );
+        $itemShouldExistString = $itemShouldExist? '' : 'n\'t';
```
Comment (mnocon) [inline src/lib/Behat/BusinessContext/ContentViewContext.php]: Please add space before `?`

**Comment by lserwatka** [comment]
Could you merge it up?

**Comment by katarzynazawada** [comment]
Done (https://github.com/ezsystems/ezplatform-admin-ui/commit/f022bb9d32abb02956e018d05e3266852d35a82a)

**Pull request merged**